src/Connections/GithubRepo.ts

```diff
@@ -907,7 +907,7 @@ export class GitHubRepoConnection extends CommandConnection<GitHubRepoConnection
     }
 
     public async onIssueCommentCreated(event: IssueCommentCreatedEvent) {
-        if (this.hookFilter.shouldSkip('issue.comment.created', 'issue.comment', 'issue') || !this.matchesLabelFilter(event.issue)) {
+        if (this.hookFilter.shouldSkip('issue.comment.created', 'issue.comment') || !this.matchesLabelFilter(event.issue)) {
```

Comment:
for ref, these are at different levels

![image](https://github.com/matrix-org/matrix-hookshot/assets/2072976/8f049e41-02ca-48e8-ab14-78c751bbb914)